### PR TITLE
New CSS Module: CSS color adjustment

### PR DESCRIPTION
**Pull request by estelle**
This module was not previously documented.

### CODE REVIEW

File: files/en-us/web/css/css_color_adjustment/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - {{cssxref("color")}}
+  - {{cssxref("color-scheme")}}
+  - {{cssxref("column-rule-color")}}
+  - {{cssxref("fill")}}
+  - {{cssxref("flood-color")}}
+  - {{cssxref("lighting-color")}}
+  - {{cssxref("outline-color")}}
+  - {{cssxref("scrollbar-color")}}
+  - {{cssxref("stop-color")}}
+  - {{cssxref("stroke")}}
```
Comment (teoli2003) [inline files/en-us/web/css/css_color_adjustment/index.md:46]: These are SVG attributes used as CSS properties. We document them under SVG/Atributes (not sure if it the right way to do in the long term, but that's the current usage), so we cannot use `{{cssxref}}`. Yari automagically fix them and raises a flaw.
```suggestion
  - [`fill`](/en-US/docs/Web/SVG/Attribute/fill)
  - [`flood-color`](/en-US/docs/Web/SVG/Attribute/flood-color)
  - [`lighting-color`](/en-US/docs/Web/SVG/Attribute/lighting-color)
  - {{cssxref("outline-color")}}
  - {{cssxref("scrollbar-color")}}
  - [`stop-color`](/en-US/docs/Web/SVG/Attribute/stop-color)
  - [`stroke`](/en-US/docs/Web/SVG/Attribute/stroke)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I made some adjustments, fixing the flaws that Yari is reporting. One question is still open, though.

File: files/en-us/web/css/css_color_adjustment/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: CSS Color Adjustment
```
Comment (dipikabh) [inline files/en-us/web/css/css_color_adjustment/index.md:2]:
```suggestion
title: CSS color adjustment
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Just one comment :)

File: files/en-us/web/css/css_color_adjustment/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+The **CSS color adjustment** module provides a model and controls over automatic color adjustment by the user agent to handle user preferences, such as "Dark Mode", contrast adjustment, and other color scheme preferences.
+
+Together with the {{CSSxRef("@media")}} features {{cssxref("@media/prefers-color-scheme", "prefers-color-scheme")}}, {{cssxref("@media/prefers-contrast", "prefers-contrast")}} and {{cssxref("@media/forced-colors", "forced-colors")}}, this module defines how and when colors are automatically adjusted by the browser.
+
+## Reference
+
+### Properties
+
+- {{cssxref("color-scheme")}}
+- {{cssxref("color-adjust")}} shorthand
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/css_color_adjustment/index.md:19]: i did notice this. we don't have a color adjust page yet it seems. I was going to create the page in a different PR

File: files/en-us/web/css/css_color_adjustment/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/css/css_color_adjustment/index.md:46]: Thank you. I didn't notice that!

File: files/en-us/web/css/css_color_adjustment/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (teoli2003) [reply files/en-us/web/css/css_color_adjustment/index.md:19]: 👍

**Comment by teoli2003** [comment]
> Just one comment :)

We need to fix slugs, titles, and links to them for all modules. It is impredictable right now, which makes authoring painful.

**Review by teoli2003 — APPROVED** [review]

**Review by dipikabh — APPROVED** [review]
Thanks!

**Pull request merged**